install.sh

```diff
@@ -187,5 +200,30 @@ function build_codemirror {
 
 run $BUILD/CodeMirror build_codemirror
 
+# Install Android SDK
+
+run $DOWNLOADS        wget https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip
```

Comment:
This worries me.  Is this going to change every few weeks, so that the install script never actually works right?  Does Google have a policy for how long Android SDK releases will remain available?